README.md

```diff
@@ -48,6 +48,10 @@ let g:neoformat_enabled_erlang = ['steamroller']
 autocmd BufWritePre rebar.config,*.[he]rl,*.app.src Neoformat steamroller
 ```
 
+### Visual Studio Code
+
+You can use steamroller from Visual Studio Code with the [Erlang Formatter](https://marketplace.visualstudio.com/items?itemName=szTheory.erlang-formatter) extension.
```

Comment:
```suggestion
You can use steamroller from Visual Studio Code with the [Erlang Formatter](https://marketplace.visualstudio.com/items?itemName=szTheory.erlang-formatter) extension created by [szTheory](https://github.com/szTheory).
```

I think it's completely reasonable to give yourself some credit here :)

Thanks a lot for your work! Really cool.